pallets/automation-time/src/mock.rs

```diff
@@ -50,6 +50,9 @@ pub const NATIVE: CurrencyId = 0;
 pub const NATIVE_LOCATION: MultiLocation = MultiLocation { parents: 0, interior: Here };
 pub const FOREIGN_CURRENCY_ID: CurrencyId = 1;
 
+pub const FEE_PER_SECOND: u128 = 100_000_000_000;
+pub const XCMP_TASK_REF_TIME: u128 = 20_000;
```

Comment:
You need to map the fee_per_second to destination, so we have 

dest1: {
    weight,
    refTime,
    assets: [ {token 1, fee per second 1}, {token 2, fee per second 2} ]
}

dest2 ..

and test them.